qa/Rakefile

```diff
@@ -24,10 +30,10 @@ namespace :qa do
     desc "Bootstrap all the VM's used for this tests"
     task :setup, :platform do |t, args|
       config   = PlatformConfig.new
-      machines = config.select_names_for(args[:platform])
+      experimental = (ENV['LS_QA_EXPERIMENTAL_OS'].to_s.downcase || "false") == "true"
+      machines = config.select_names_for(args[:platform], {"experimental" => experimental})
```

Comment:
We are using rake task arguments and an environment variable in the same task can we consolidate in only one way?
Since we are using rake arguments I think we should use that for the experimental option? Opinions?